crowdfunding_compassion/controllers/donation_controller.py

```diff
@@ -87,9 +87,14 @@ def post_donation_form(self, project, participant, amount, **post):
             sale_order = request.website.sale_get_order(force_create=True).sudo()
         product = project.product_id
         quantity = float(amount) / product.standard_price
+        price = product.standard_price
+        if not quantity.is_integer():
+            quantity = 1
+            price = amount
+
```

Comment:
This can be done regardless if the quantity is entire or not.
```suggestion
        quantity = 1
        price = amount
```
PS: the line above can also be removed.

Reply:
But wouldn't it be weird if after selecting `3 toilet accesses` it shows `toilet accesses 1` for 90.- instead of `toilet access 3`? Or maybe also change the name of the product to `3 toilet accesses` directly?
My change:
![image](https://github.com/user-attachments/assets/bbd4ddb9-6868-4f5e-909a-47ae5f22cf2d)
Your proposition:
![image](https://github.com/user-attachments/assets/7fb9a8f6-0179-49e6-a44e-028d9890ab7a)
New Proposition:
![image](https://github.com/user-attachments/assets/49051435-eb10-45e8-95a0-35cd7ecb476c)
And maybe something like that for custom donations
![image](https://github.com/user-attachments/assets/2d677ee0-e5ec-4559-9500-a156b6f28401)

Reply:
You are right, your changes make more sense :)